dashboard/test/ui/features/step_mode.feature

```diff
@@ -10,7 +10,8 @@ Scenario: Step Only - Failure
 
   When I drag block "1" to block "5"
     And I press "stepButton"
-    And I wait until "#stepButton" is enabled
+    And I wait up to 1 second until "#stepButton" is disabled
+    And I wait until "#stepButton" is not disabled
```

Comment:
I think I see what you are doing here, but it is not super intuitive from reading these steps. I think you are dealing with this situation:
* button is initially enabled
* button becomes disabled and then enabled again
* wait for it to be enabled, making sure it goes through the disabled state first, allowing for the fact that it may go enabled --> disabled --> enabled before our any of our test steps have time to run on it

I think it would improve readability to insert the word `or` inside the optional `wait` clause, so you would have: `I wait up to 1 second or until "#stepButton" is disabled`

Alternately, if we're only ever waiting 1 second, it could be fine just to `wait for 1 second` and not worry about saving a fractional second if the button becomes disabled during that time.

Reply:
Thanks for the feedback on this- I took your suggestion and decided to go with `wait for 1 second` to keep things simple.